src/drive/mobile/lib/reporter.js

```diff
@@ -1,21 +1,19 @@
 /* global __SENTRY_TOKEN__, __DEVELOPMENT__ */
 import Raven from 'raven-js'
 
-let isEnabled = false
 const PROD_ANALYTICS_URL = `https://${__SENTRY_TOKEN__}@sentry.cozycloud.cc/6`
 const DEV_ANALYTICS_URL = `https://${__SENTRY_TOKEN__}@sentry.cozycloud.cc/2`
 export const ANALYTICS_URL = __DEVELOPMENT__
   ? DEV_ANALYTICS_URL
   : PROD_ANALYTICS_URL
 
-export const getReporterConfiguration = () => ({
+export const getReporterConfiguration = (isEnabled = false) => ({
```

Comment:
Another possible problem, I haven't checked though : this function is called elsewhere, and without parameters. The new net effect is that the reporter will be globally disabled, which we really don't want. (I may be wrong on this one, but really, this whole reported code is fragile and I recommend not touching it as part of a larger work).